File: NutritionControl.DTO/Models/ProductDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NutritionControl.DTO.Models
{
    public class ProductDto
    { 
        public int Id { get; set; }
        public string Name { get; set; }
        public string PhotoUrl { get; set; }
        public decimal CaloriesValue { get; set; }
        public decimal Protein { get; set; }
        public decimal Fats { get; set; }
        public decimal Carbohydrates { get; set; }
        public string CategoryName { get; set; }
    }
}

File: NutritionControl.Client/client-app/src/app/models/productDto.ts
export class ProductDto  { 
  id: number;
  name: string;
  caloriesValue: number;
  protein: number;
  fats: number;
  carbohydrates: number;
  categoryName: string;
  isLiked: boolean;
}

export class GroupedProducts {
  categoryName: string;
  isCollapsed: boolean = true;
  products : Array<ProductDto>;
}